psukhotski/Lesson6/Lesson6_DZadd.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html> 
+<HTML> 
+  <HEAD>
+		<META charset="utf-8">
+		<TITLE>Адаптивное меню</TITLE>
+		<style type="text/css">
+			ul {
+        list-style: none;
+        margin: 0;
+        padding: 0;
+        border-bottom: 1px solid transparent;
+        border-radius: 10px;
+        width: 150px;        
+      }
+      li {
+        position: relative;
+      }
+      li ul {
+        display: none;
+        position: absolute;
+        top: 0;
+        left: 150px;
+        opacity: 0;
+      }
+      li:hover ul {
+        display: block;
+        opacity: 1;     /*мне показалось, что с opacity верное решение, но почему-то не заработало*/
```

Comment:
if you use opacity, then change display: none (in line 19) to block, and in line 28 replace all to opacity